web/server/codechecker_server/server.py

```diff
@@ -444,19 +454,7 @@ def do_POST(self):
             LOG.warning(str(exn))
             import traceback
             traceback.print_exc()
-            ex = TApplicationException(TApplicationException.INTERNAL_ERROR,
-                                       str(exn))
-            fname, _, seqid = iprot.readMessageBegin()
-            oprot.writeMessageBegin(fname, TMessageType.EXCEPTION, seqid)
-            ex.write(oprot)
-            oprot.writeMessageEnd()
-            oprot.trans.flush()
-            result = otrans.getvalue()
-            self.send_response(200)
-            self.send_header("content-type", "application/x-thrift")
-            self.send_header("Content-Length", len(result))
-            self.end_headers()
-            self.wfile.write(result)
+            self.send_exception(str(exn), iprot, oprot, otrans)
```

Comment:
Please move the comment in the Exception handling part to the new `send_exception` function. The conversion is done there.